site/docs/components/static-list/examples.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+  <LivePreview componentName="static-list" exampleName="Multiple" >
+
+## Multiple
```

Comment:
Is this name correct still?

Reply:
Wondering if "Default" describes better the example rather than using "Multiple".